src/collectionData.js

```diff
@@ -0,0 +1,128 @@
+export default class TodoListData {
+  constructor() {
+    this.todoListTasks = JSON.parse(window.localStorage.getItem('todoList')) || [];
+    this.taskRemoveButtons = [];
+  }
+
+  addTask = (task) => {
+    task.taskIndex = this.todoListTasks.length + 1;
+    this.todoListTasks.push(task);
+    this.updateStorage();
+  }
+
+  removeTask = (index) => {
+    this.todoListTasks = this.todoListTasks.filter((task) => task.taskIndex !== index);
+    this.todoListTasks.forEach((task, i) => {
+      task.taskIndex = i + 1;
```

Comment:
- Please try to avoid repetition of a code. It would make your project unprofessional and miss the DRY rule.

Reply:
Done! thanks for pointing that out.